src/main/java/com/ft/methodearticlemapper/transformation/EomFileProcessor.java

```diff
@@ -101,13 +102,13 @@ public Content processPreview(EomFile eomFile, String transactionId) {
         UUID uuid = UUID.fromString(eomFile.getUuid());
 
         PublishEligibilityChecker eligibilityChecker =
-            PublishEligibilityChecker.forEomFile(eomFile, uuid, transactionId);
+                PublishEligibilityChecker.forEomFile(eomFile, uuid, transactionId);
```

Comment:
They are continuously improving it, so we might want to take a look at the latest version:
https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml.

The one I shared is at least half year old, but probably more.